### ISSUE TITLE
build scripts call zsh, should be just sh

### ISSUE THREAD

**Issue by JoshuaPettus** [shinyquagsire23/OpenJKDF2#240]
It took me longer than it should to have figured this out.  When I was trying to run your build compile scripts on fedora linux, it kept saying Script error, "cannot execute: required file not found."  My first thought is that these were in a dos format and not a unix format, but converting it did nothing.  Then i finally looked into the script and saw that the shebang said #!/bin/zsh.  Now I'm a proponent of zsh as much as the next person, but it's really not the standard anywhere, nor is it necessary.  A generic /bin/sh will work now matter what shell is being used.

**Comment by shinyquagsire23** [comment]
I'll be honest, part of that was because macOS has *really* crusty old versions for bash/sh, and zsh is the only thing that's both recent and wouldn't have multiple versions on the filesystem. And I wasn't sure if the `if [ ! -f whatever ]; then` lines would hit that crustiness. idk though, when I tracked down the `byacc` version from 1995 the cogscripts used, I definitely hit some issues with if statements.

It does seem like everything works with `#!/bin/sh` though so, ehh. Fixed.
